app/config/translations.json

```diff
@@ -1,179 +1 @@
-{
```

Comment:
Let's make sure the file is formatted with indentation and sorted keys, so that changes are tracked more easily.